src/python/bot/fuzzers/syzkaller/runner.py

```diff
@@ -73,48 +75,76 @@ def _create_empty_testcase_file(self):
+      repro_args: A sequence of arguments to be passed to the executable.
+    """
+    logs.log('Running Syzkaller Rrpro!')
+    additional_args = copy.copy(repro_args)
+    result = self.run_and_wait(additional_args, timeout=repro_timeout)
+    logs.log('Syzkaller Repro Stopped!')
+    return engine.ReproduceResult(result.command, result.return_code,
+                                  result.time_executed, str(result.output))
 
   def fuzz(self,
            fuzz_timeout,
            additional_args,
            unused_additional_args=None,
            unused_extra_env=None):
-    """This is where actual syzkaller fuzzing is done."""
+    """This is where actual syzkaller fuzzing is done.
+    Args:
+      fuzz_timeout: The maximum time in seconds that fuzz job is allowed
+          to run for.
+      additional_args: A sequence of additional arguments to be passed to
+          the executable.
+    """
+    logs.log('Running Syzkaller!')
     additional_args = copy.copy(additional_args)
     fuzz_result = self.run_and_wait(additional_args, timeout=fuzz_timeout)
-
-    log_lines = utils.decode_to_unicode(fuzz_result.output).splitlines()
-    fuzz_result.output = None
-    crash_testcase_file_path = self.get_testcase_path(log_lines)
-
-    #TODO(hzawawy): remove once syzkaller code is completed.
-    if not crash_testcase_file_path and fuzz_result.return_code:
-      crash_testcase_file_path = self._create_empty_testcase_file()
-
-    fuzz_logs = '\n'.join(log_lines)
-
-    # TODO(hzawawy): Parse stats information and add them to FuzzResult
-    parsed_stats = []
-
-    crashes = []
-    if crash_testcase_file_path:
-      #TODO(hzawawy): add repro arguments
-      reproduce_arguments = []
-      actual_duration = int(fuzz_result.time_executed)
-      # Write the new testcase.
-      # Copy crash testcase contents into the main testcase path.
-      crashes.append(
-          engine.Crash(crash_testcase_file_path, fuzz_logs, reproduce_arguments,
-                       actual_duration))
-
-    return engine.FuzzResult(fuzz_logs, fuzz_result.command, crashes,
-                             parsed_stats, fuzz_result.time_executed)
+    logs.log('{}{}'.format('Syzkaller Stopped! Fuzzing timed out: ',
+                           fuzz_timeout))
+    files = {}
+    fuzz_logs = ''
+
+    logs.log('{}{}'.format('Current time: ', time.time()))
+    rootdir = constants.SYZKALLER_WORK_FOLDER
+    pattern = 'report*'
+    visited = set()
+    for subdir, _, files in os.walk(rootdir):
+      for file in files:
+        subdir_file = subdir[2:] + file
+        if fnmatch.fnmatch(file, pattern) and subdir_file not in visited:
+          visited.add(subdir_file)
+          f = open(os.path.join(subdir, file), 'r')
+          log_lines = (f.read())
+          fuzz_result.output = str(log_lines)
+          f.close()
+
+          # Get the log corresponding to this specitic crash
+          crash_testcase_file_path = os.path.join(subdir, 'log' + file[6:])
```

Comment:
what is 6:, can you please add comment with example. any other cleaner way to do this ?